addons/mail/models/mail_ice_server.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+import requests
+import json
+
+
+class MailIceServer(models.Model):
```

Comment:
It's a copy-paste of the file in the branch https://github.com/odoo-dev/odoo/tree/master-mail-rtc-tso
It would have more sense to have the names and the file location being agnostic of the module because two modules will use this configuration (discuss and web_editor).